src/features/post/components/Post.tsx

```diff
@@ -33,7 +35,6 @@ export const Post: FC<PostProps> = ({post}: PostProps) => {
   const reactToPost = async (pid: string, reactionType: ReactionType) => {
```

Comment:
reactions is still not updated in this case
move reactions state in this component and update it here